syntax = "proto3";

package teos;

import "appointment.proto";
import "user.proto";

service HTTP_API {
  rpc register(RegisterRequest) returns (RegisterResponse) {}
  rpc add_appointment(AddAppointmentRequest) returns (AddAppointmentResponse) {}
  rpc get_appointment(GetAppointmentRequest) returns (GetAppointmentResponse) {}
}